config/management/src/secure_backend.rs

```diff
@@ -152,13 +153,21 @@ pair: "k0=v0;k1=v1;...".  The current supported formats are:
     }
 }
 
+// "ignore" allows this to be used elsewhere to provide for greater richness when required
```

Comment:
I'm not quite sure what the greater richness is?

Reply:
look at keys.rs, this allows us to pull from disk directly without having to make a fake storage

Reply:
I'm still confused by this comment 😕

Reply:
hidden within the macro itself now